x/evm/evmtest/test_deps.go

```diff
@@ -27,7 +27,7 @@ type TestDeps struct {
 	Sender   EthPrivKeyAcc
 }
 
-func (deps TestDeps) GoCtx() context.Context {
+func (deps *TestDeps) GoCtx() context.Context {
```

Comment:
I would leave this as a non-pointer receiver to indicate to the caller that it does not modify the underlying `TestDeps` object. 

```suggestion
func (deps TestDeps) GoCtx() context.Context {
```